org.sil.FieldWorks.yml

```diff
@@ -0,0 +1,787 @@
+# FieldWorks Flatpak package build specification
+#
+# Copyright (c) 2021 SIL International. MIT License.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+app-id: org.sil.FieldWorks
+
+# org.gnome.Platform provides some pieces that we would need to manually specify
+# if we use org.freedesktop.Platform.
+runtime: org.gnome.Platform
+# Target a runtime that is most similar to Ubuntu 20.04, for which we can easily
+# build and test on without flatpak.
+runtime-version: "3.36"
+sdk: org.gnome.Sdk
+
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  # FieldWorks can synchronize project data over the network with other users.
+  - --share=network
+  # FieldWorks allows playing sound files and playing and recording in audio
+  # writing systems.
+  - --socket=pulseaudio
+  # FieldWorks can import language and dictionary data, restore from backup
+  # files, and link to pictures or other media to appear in a dictionary.
+  # Users need to save project data, project backups, Encoding Converter
+  # configurations, and configuration and settings of FieldWorks and included
+  # components.
+  # Users need to be able to export dictionaries to arbitrary locations.
+  - --filesystem=host
```

Comment:
why would it need to write outside 'home' ?

Reply:
I was thinking if I limited it to home, users wouldn't be able to export to a USB Flash Drive.